### PR TITLE
Improvements for Docker usage

### PR DESCRIPTION
**Pull request by bebehei**
Small improvement for the dockerfile to match best practices and a small improvement to make the low level docker call unnecessary.

### CODE REVIEW

File: contrib/docker/README.md [hunk 1]
```diff
@@ -9,13 +9,7 @@ use that server.
...
 ## Build
 
-Build the docker image with the `docker build` command from the root of the synapse repository.
-
-```
-docker build -t docker.io/matrixdotorg/synapse .
-```
-
-The `-t` option sets the image tag. Official images are tagged `matrixdotorg/synapse:<version>` where `<version>` is the same as the release tag in the synapse git repository.
+Build the docker image with the `docker-compose build` command.
```
Comment (jcgruenhage) [inline contrib/docker/README.md]: perhaps we should document that people can skip building and specify the version where we have `latest` in the docker-compose file, to pull it from docker hub.

Comment (bebehei) [reply contrib/docker/README.md]: Yes, full ACK. But my intention is to fix this later in a separate PR. This is my first one and I wanted to get the PR process known first.

**Comment by bebehei** [comment]
Hey guys, what's missing in this PR to get finally merged?

**Comment by jcgruenhage** [comment]
Nothing as far as I can tell. @richvdh, was I supposed to merge this after approving it? I guess I did something wrong and this fell off the radar because of that.

**Pull request merged**